docs/usage.md

```diff
@@ -438,6 +438,17 @@ nextflow run nf-core/proteinfold \
       -profile <docker>
 ```
 
+```bash
```

Comment:
I think `console` is for commands and `bash` for scripts, maybe we could include `$ ` before the command to make it more clear?
```suggestion
```console
```